ci/build/release-prep.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Description: This is a script to make the release process easier
+# Run it with `yarn release:prep` and it will do the following:
+# 1. Check that you have a $GITHUB_TOKEN set and hub installed
+# 2. Update the version of code-server (package.json, docs, etc.)
+# 3. Update the code coverage badge in the README
+# 4. Open a draft PR using the release_template.md and view in browser
+# If you want to perform a dry run of this script run DRY_RUN=0 yarn release:prep
+
+set -euo pipefail
+
+main() {
+  if [ "${DRY_RUN-1}" -eq 0 ]; then
+    echo "Performing a dry run..."
+    # Disabled because shellcheck wants me to do this
+    # SC2209: Use var=$(command) to assign output (or quote to assign string).
+    # But neither of those match my needs
+    # We do this so that the dry run echos the commands instead of running them
+    # shellcheck disable=SC2209
+    CMD=echo
+  else
+    CMD=''
+  fi
+
+  cd "$(dirname "$0")/../.."
+
+  # Check that $GITHUB_TOKEN is set
+  if [[ -z "${GITHUB_TOKEN}" ]]; then
+    echo "We couldn't find an environment variable under GITHUB_TOKEN."
+    echo "This is needed for our scripts that use hub."
+    echo -e "See docs regarding GITHUB_TOKEN here under 'GitHub OAuth authentication': https://hub.github.com/hub.1.html"
+    exit
+  fi
+
+  # Check that hub is installed
+  if ! command -v hub &>/dev/null; then
+    echo "hub could not be found."
+    echo "We use this with the release-github-draft.sh and release-github-assets.sh scripts."
+    echo -e "See docs here: https://github.com/github/hub#installation"
+    exit
+  fi
+
+  # Check that they have jq installed
+  if ! command -v jq &>/dev/null; then
+    echo "jq could not be found."
+    echo "We use this to parse the package.json and grab the current version of code-server."
+    echo -e "See docs here: https://stedolan.github.io/jq/download/"
+    exit
+  fi
+
+  # Check that they have rg installed
+  if ! command -v rg &>/dev/null; then
+    echo "rg could not be found."
+    echo "We use this when updating files across the codebase."
+    echo -e "See docs here: https://github.com/BurntSushi/ripgrep#installation"
+    exit
+  fi
+
+  # Check that they have sd installed
+  if ! command -v sd &>/dev/null; then
+    echo "sd could not be found."
+    echo "We use this when updating files across the codebase."
+    echo -e "See docs here: https://github.com/chmln/sd#installation"
+    exit
+  fi
+
+  # Check that they have node installed
+  if ! command -v node &>/dev/null; then
+    echo "node could not be found."
+    echo "That's surprising..."
```

Comment:
Ahahahahaha true true :laughing:

Reply:
I should put a link to that Halo 3 image 🤣